demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -312,6 +312,22 @@ def test_get_input_data_complex():
     assert _value == 'File.Name'
 
 
+def test_generate_playbook(playbook):
+    from demisto_sdk.commands.generate_docs.generate_playbook_doc import \
+        generate_playbook_doc
+
+    playbook.create_default_playbook()
```

Comment:
remove this. When you're using "write_dict" you're overwritting the default playbook.